internal/config/config.go

```diff
@@ -24,6 +24,8 @@ import (
 )
 
 const (
+	// BuildRequest is a JSON file that describes which library to build/test.
+	BuildRequest      string = "build-request.json"
```

Comment:
Question, what is different in content between "build-request.json" to "generate-request.json"?

Reply:
Looking at the two examples in go/librarian:language-onboarding, I don't see any difference.

Reply:
fyi - I asked similar question [here](https://docs.google.com/document/d/1bQ5aXIfQEauI6cHlZKh8R6PBAG_sfHRaDxHcxRCQMYI/edit?resourcekey=0-TyGOVF1t5hGtF0Pydz572w&disco=AAABk_vU9oc). We should keep the current names for Java/Go onboarding.